stdlib/test/collections/test_dict.mojo

```diff
@@ -479,10 +479,19 @@ def test_find_get():
     assert_equal(some_dict.get("not_key", 0), 0)
 
 
+fn test_clear() raises:
```

Comment:
Perhaps we could also check that `clear` works on an empty dictionary? That would be a useful edge case to check.